yellowbrick/text/postag.py

```diff
@@ -114,252 +131,279 @@ def __init__(
                     elif tag in ["DT" or "PDT"]:
-                        self.pos_tag_counts_["determiner"] += 1
+                        counter["determiner"] += 1
                     elif tag == "EX":
-                        self.pos_tag_counts_["existential"] += 1
+                        counter["existential"] += 1
                     elif tag == "FW":
-                        self.pos_tag_counts_["non-English"] += 1
+                        counter["non-English"] += 1
                     elif tag == "IN":
-                        self.pos_tag_counts_["preposition"] += 1
+                        counter["preposition"] += 1
                     elif tag == "POS":
-                        self.pos_tag_counts_["possessive"] += 1
+                        counter["possessive"] += 1
                     elif tag == "LS":
-                        self.pos_tag_counts_["list"] += 1
+                        counter["list"] += 1
                     elif tag == "MD":
-                        self.pos_tag_counts_["modal"] += 1
+                        counter["modal"] += 1
                     elif tag in self.punct_tags:
-                        self.pos_tag_counts_["punctuation"] += 1
+                        counter["punctuation"] += 1
                     elif tag == "TO":
-                        self.pos_tag_counts_["infinitive"] += 1
+                        counter["infinitive"] += 1
                     elif tag == "UH":
-                        self.pos_tag_counts_["interjection"] += 1
+                        counter["interjection"] += 1
                     elif tag == "SYM":
-                        self.pos_tag_counts_["symbol"] += 1
+                        counter["symbol"] += 1
                     else:
-                        self.pos_tag_counts_["other"] += 1
-
+                        counter["other"] += 1
+        
     def draw(self, **kwargs):
         """
         Called from the fit method, this method creates the canvas and
         draws the part-of-speech tag mapping as a bar chart.
-
+        
         Parameters
         ----------
         kwargs: dict
             generic keyword arguments.
-
+        
         Returns
         -------
         ax : matplotlib axes
             Axes on which the PosTagVisualizer was drawn.
         """
         colors = resolve_colors(
-            n_colors=len(self.pos_tag_counts_),
-            colormap=self.colormap,
-            colors=self.colors,
-        )
-
+                    n_colors=len(self.pos_tag_counts_), 
+                    colormap=self.colormap,
+                    colors=self.colors
+                    )
+        
+        pos_tag_sum = copy.deepcopy(self.pos_tag_counts_)
+        try:
+            #Stores the tag-wise sum of each label
+            pos_tag_sum = sum((Counter(i) for i in self.pos_tag_counts_.values())
+                            , Counter())
+        except TypeError:
+            pass
```

Comment:
This is interesting - I do like the EAFP (easier to ask forgiveness than permission) over the LYBL (look before you leap) style of more pythonic programming. I suppose the idea here is that this is only used for `stack=True` and an exception will be raised if the type is not correct in the `stack=False` case? 

Although this line of code will likely change when you create the stack helper function -- I would like to point out two things:

1. When doing something like this - it is important to add comments! We want to make sure that another contributor and future you (in 3 months time) can understand what's happening in this code! 
2. Line 323 represents a safety flaw - although doing nothing is what you're going for here, this try/except could capture a different type of error, burying it and making bugs hard to diagnose. If you did `pos_tag_sum=None` instead of `pass` - it would at least allow `None` type errors later in the program which would guide us to this line of code.